MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/text/LocalGlyphRasterizer.java

```diff
@@ -31,7 +31,7 @@ of the bitmap (y: 20) with some buffer around the edge
 
     paint = new Paint();
     paint.setAntiAlias(true);
-    paint.setTextSize(24);
```

Comment:
@Chaoba All locally and server generated fonts are rendered at 24 points. Maybe you need to increase bitmap size. Issue needs bit more investigation, 24pt glyph should fit into 35x35 bitmap.

Reply:
Increate bitmap size will twist characters.